dev/Repeater/TestUI/Samples/FlexboxAlignContentPage.xaml

```diff
@@ -0,0 +1,135 @@
+﻿<Page
```

Comment:
I'm not seeing a test page for flex basis. We'll want to add a page specifically for that property even if it is being used in other pages.